src/avt/VisWindow/Colleagues/VisWinRendering.C

```diff
@@ -545,6 +545,9 @@ VisWinRendering::EnableDepthPeeling()
 //  Creation:   Wed Aug 12 11:49:37 PDT 2015
 //
 //  Modifications:
+//    Eric Brugger, Tue Jun 13 17:25:05 PDT 2023
+//    Remove the call enabling multi sampling. This fixes a bug where the
+//    visualization window is black when using mesagl.
```

Comment:
I tested this with the VTK-9-built-with-system-gl on toss4, and it fixes the grey background window too.

Will this change cause bad behavior with VTK-8? Should it be conditional on VTK version?

Also, if the use of multiSamples is to be removed here, should it be removed completely from this class?

Reply:
It will affect VTK-8 behavior since on some systems we do get some anti-aliasing. With this change we wouldn't get AA anywhere. I will make the change conditional on VTK-9 to preserve the current behavior.

Reply:
> Also, if the use of multiSamples is to be removed here, should it be removed completely from this class?

For VTK 9, should it be removed completely from this class (there are other places where multiSamples var is used) ?